webapp/src/cloud.ts

```diff
@@ -161,13 +161,15 @@ function resetAsync(): Promise<void> {
     return Promise.resolve();
 }
 
-let _inProgressSyncAsync: Promise<any> = Promise.resolve()
+let inProgressSyncPromise: Promise<any> = Promise.resolve()
 export async function syncAsync(hdrs?: Header[]): Promise<Header[]> {
```

Comment:
>export [](start = 0, length = 6)

Does this function need to be exported? Seems dangerous.

Reply:
It should be safe to use liberally, it'll just hit the cloud a lot. And today it is called rather aggressively...

Ideally this should be called sparingly and probably use a similar debounce logic as our file saving. TODO :)